optimum/exporters/onnx/model_configs.py

```diff
@@ -1160,6 +1154,171 @@ def outputs(self) -> Dict[str, Dict[int, str]]:
         }
 
 
+class T5EncoderOnnxConfig(CLIPTextOnnxConfig):
+    @property
+    def inputs(self):
+        return {
+            "input_ids": {0: "batch_size", 1: "sequence_length"},
+        }
+
+    @property
+    def outputs(self):
+        return {
+            "last_hidden_state": {0: "batch_size", 1: "sequence_length"},
+        }
+
+
+class DummyTransformerTimestpsInputGenerator(DummyTimestepInputGenerator):
```

Comment:
Why don't you put these dummy input generators under the inputs generator file?

Reply:
I thought that's what we do when  the dummy input generator is model-specific, will move them 👍